modules/images/dominant-color/load.php

```diff
@@ -126,9 +129,10 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 		$extra_class  = $image_meta['has_transparency'] ? 'has-transparency' : 'not-transparent';
 	}
 
-	if ( ! empty( $data ) || ! empty( $style ) ) {
-		$filtered_image = str_replace( '<img ', '<img ' . $data . $style, $filtered_image );
+	if ( ! empty( $data ) ) {
+		$filtered_image = str_replace( '<img ', '<img ' . $data, $filtered_image );
 	}
+
```

Comment:
Nice, I like a lot how this PR separates the logic for adding `$data` from adding `$style` to the image tag. This makes it much easier to follow the code here.

Reply:
👍